janusgraph-driver/src/main/java/org/janusgraph/graphdb/tinkerpop/io/binary/RelationIdentifierGraphBinarySerializer.java

```diff
@@ -40,6 +40,10 @@ protected void writeNonNullableValue(RelationIdentifier value, Buffer buffer, Gr
         buffer.writeLong(((Number) value.getOutVertexId()).longValue());
         buffer.writeLong(value.getTypeId());
         buffer.writeLong(value.getRelationId());
-        buffer.writeLong(((Number) value.getInVertexId()).longValue());
+        if (value.getInVertexId() != null) {
+            buffer.writeLong(((Number) value.getInVertexId()).longValue());
+        } else {
+            buffer.writeLong(0);
```

Comment:
Pretty sure this branch is covered by https://github.com/JanusGraph/janusgraph/actions/runs/3856024161/jobs/6571765068#step:6:310 but no idea why this is not captured by Codecov.